torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -466,6 +447,15 @@ ExecutionPlan ProfilingGraphExecutorImpl::getPlanFor(
     return *optimized_plan_;
   }
 
+  // no opt mode
+  if (!getGraphExecutorOptimize()) {
+    auto copy = graph->copy();
```

Comment:
This has the opposite problem now - if you reenable optimization after running it with disabling, you will never optimize.

Reply:
err i'm kinda thinking it should be okay. If the user changes the number of profiled runs midway, we won't update it for models in flight either.

Reply:
Alternatively, we could try disregarding `getGraphExecutorOptimize` we are already profiling but this seems complicated conceptually